icu4c/source/i18n/measunit_extra.cpp

```diff
@@ -415,6 +415,11 @@ UBool U_CALLCONV cleanupUnitExtras() {
     gSimpleUnitCategories = nullptr;
     uprv_free(gSimpleUnits);
     gSimpleUnits = nullptr;
+    uprv_free((void*)gUnitReplacementStrings);
```

Comment:
nit: we should not need this cast

Reply:
We _shouldn't_, but my compiler (for some reason) yelled at me when I didn't have it.